packages/sensor/example/lib/main.dart

```diff
@@ -0,0 +1,115 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'package:flutter/material.dart';
+import 'package:sensor/sensor.dart';
+
+import 'snake.dart';
+
+void main() {
+  runApp(new MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return new MaterialApp(
+      title: 'Sensor Demo',
+      theme: new ThemeData(
+        primarySwatch: Colors.blue,
+      ),
+      home: new MyHomePage(title: 'Flutter Demo Home Page'),
+    );
+  }
+}
+
+class MyHomePage extends StatefulWidget {
+  final String title;
+
+  MyHomePage({Key key, this.title}) : super(key: key);
+
+  @override
+  _MyHomePageState createState() => new _MyHomePageState();
+}
+
+class _MyHomePageState extends State<MyHomePage> {
+  List<double> _accelerometerValues;
+  List<double> _gyroscopeValues;
+  List<StreamSubscription<List<double>>> _streamSubscriptions =
+      <StreamSubscription<List<double>>>[];
+
+  @override
+  Widget build(BuildContext context) {
+    List<String> accelerometer =
+        _accelerometerValues?.map((double v) => v.toStringAsFixed(1))?.toList();
+    List<String> gyroscope =
+        _gyroscopeValues?.map((double v) => v.toStringAsFixed(1))?.toList();
+
+    return new Scaffold(
+      appBar: new AppBar(
+        title: const Text('Sensor Example'),
+      ),
+      body: new Column(
+        mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+        children: <Widget>[
+          new Center(
+            child: new DecoratedBox(
+              decoration: new BoxDecoration(
+                border: new Border.all(width: 1.0, color: Colors.black38),
+              ),
+              child: new SizedBox(
+                height: 200.0,
+                width: 200.0,
+                child: new Snake(),
```

Comment:
The magic constant `200.0` is implicitly used inside the Snake implementation where it appears as `20` and `10.0`. Such covert coupling makes the code hard to experiment with for whoever wants to play with it.

Can't we refactor that to something like `child: new Snake(rows: 20, columns: 20, size: 10.0)` and drop the `SizedBox` here?